phoenix-core/src/main/java/org/apache/phoenix/schema/PTable.java

```diff
@@ -583,6 +583,7 @@ private static int getReservedQualifier(byte[] bytes, int offset, int length) {
     PName getName();
     PName getSchemaName();
     PName getTableName();
+    PName getPhysicalTableName();
```

Comment:
Distinction between getPhysicalTableName and getPhysicalName can be confusing

Reply:
Renamed getPhysicalTableName as getPhysicalTableNameColumnInSyscat. getPhysicalName is inferred and I could have renamed it but that would touch a lot of files.